interface/core.xhr.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "core.xhr",
+  "api": {
+    // @param {String} method - GET/POST/DELETE
+    // @param {String} url
+    // @param {Boolean} async
```

Comment:
Is there a good reason to include this?  Given that the interface is always asynchronous, I'd be inclined to remove it.